src/components/article/ImageBanner/index.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
+import Banner from "../../../assets/img/banner.jpg";
+
+export default class ImageBanner extends Component {
+  render() {
+    return (
+      <React.Fragment>
+        <section className="image-banner">
+          <img src={Banner} alt="" />
+          <div className="container">
+            <div className="banner-content text-center">
+              <h2>{this.props.info}</h2>
+              <p>
+Home
+                <i className="fa fa-angle-right" /> 
+                {" "}
+                {this.props.page}
```

Comment:
Hey @wasibani-roy please help  clarify on importance of this expression

Reply:
This just helps me put an empty space before i display the page